test/integration/shareable_preview_generate_new_link_test.rb

```diff
@@ -22,18 +22,20 @@ class ShareablePreviewGenerateNewLinkIntegrationTest < ActionDispatch::Integrati
 
       test "it revokes the previous links and generates a new one" do
         get admin_case_study_path(edition)
+        find(".govuk-details__summary").click
```

Comment:
I'm wondering if this would work and be more readable – (and more future proof if we add a second 'details' component to the page down the line)

```suggestion
        find("summary", text: "Share document preview").click
```

(and ditto for where it's used on line 33)

Reply:
yea good shout. Will update it